launcher/installation.md

```diff
@@ -4,8 +4,8 @@
 
```

Comment:
@jasonmadigan This has gotten a lot longer and more complex.
It would be good to go through all these steps from scratch and find any missing bits